src/Microsoft.AspNet.Mvc.Core/ModelBinding/PrefixContainer.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using Microsoft.Framework.Internal;
 
 namespace Microsoft.AspNet.Mvc.ModelBinding.Internal
```

Comment:
Could you move this to a `Internal` subdirectory \ remove the `Internal` namespace?